IOS/SwiftSeedProject/SwiftSeedProject/Domain/Services/PersonService.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  PersonService.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/28/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+open class PersonService {
+    let dataProvider : InMemoryDataProvider
+    
+    var persons = [Person]()
```

Comment:
Type Annotation for Empty Arrays and Dictionaries

For empty arrays and dictionaries, use type annotation. (For an array or dictionary assigned to a large, multi-line literal, use type annotation.)

Preferred:
var names: [Person] = []

Not Preferred:
var persons = [Person]()